fenzo-core/src/main/java/com/netflix/fenzo/TaskSchedulingService.java

```diff
@@ -191,13 +200,49 @@ public boolean isShutdown() {
                             }
                         }
                     }
+                    else if(pHostsAdded > 0) {
+                        logger.debug("No pseudo assignments made, looking for failures");
```

Comment:
This is all about logging at debug level, so lets wrap it with `if(logger.isDebugEnabled())`.